### PR TITLE
Fix CSP headers output

### PR DESCRIPTION
**Pull request by yeremeyev**
https://sourceclear.atlassian.net/browse/FLAT-535

### CODE REVIEW

File: assets/javascripts/discourse/models/csp-header.js.es6 [hunk 1]
```diff
@@ -4,13 +4,11 @@ let CspHeader = Discourse.Model.extend({ })
 
 CspHeader.reopenClass({
   createFromJson(json) {
+    console.log(json.value);
```
Comment (ArthurZaharov) [inline assets/javascripts/discourse/models/csp-header.js.es6]: Please remove log from here.

File: assets/javascripts/discourse/models/csp-header.js.es6 [hunk 2]
```diff
@@ -4,13 +4,11 @@ let CspHeader = Discourse.Model.extend({ })
...
     return this.create({
       name: json.name,
       value: json.value,
-      directives: json.value && _.map(json.value.split('; '), (directive) => {
-        let [name, value] = directive.split(/\s(.+)/);
-        return {name, value}
-      }),
+      directives: _(json.value.split(';')).map(String.trim).compact().value(),
```
Comment (ArthurZaharov) [inline assets/javascripts/discourse/models/csp-header.js.es6]: What if value undefined?

Comment (yeremeyev) [inline assets/javascripts/discourse/models/csp-header.js.es6]: Great catch, thanks! Just missed it.

**Comment by ArthurZaharov** [comment]
:+1:

**Pull request merged**